testing/wireguard-vanilla/APKBUILD

```diff
@@ -4,8 +4,8 @@
 # when changing _ver we *must* bump _rel
 # we must also match up _toolsrel with wireguard-tools
 _name=wireguard
-_ver=0.0.20180802
-_rel=2
+_ver=0.0.20180809
+_rel=1
```

Comment:
why 1?

Reply:
See the comment above it for why. Toggling between 1 and 2 has been the norm for nearly all of the 204 WireGuard commits. This PR here is a routine bump.